compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/DataflowAnalyzer.java

```diff
@@ -1345,6 +1345,8 @@ public void visit(BLangTypeInit typeInitExpr) {
     @Override
     public void visit(BLangTernaryExpr ternaryExpr) {
         analyzeNode(ternaryExpr.expr, env);
+        analyzeNode(ternaryExpr.thenExpr, env);
+        analyzeNode(ternaryExpr.elseExpr, env);
```

Comment:
This change was already done on the stage-slbeta4 branch. Can you send this PR (which is effectively the tests) to the stage branch too?